llama_index/indices/managed/vectara/retriever.py

```diff
@@ -33,17 +33,15 @@ class VectaraRetriever(BaseRetriever):
         n_sentences_after (int):
              number of sentences after the matched sentence to return in the node
         filter: metadata filter (if specified)
-        vectara_kwargs (dict): Additional vectara specific kwargs to pass
-            through to Vectara at query time.
-            * mmr_k: number of results to fetch for MMR, defaults to 50
-            * mmr_diversity_bias: number between 0 and 1 that determines the degree
-                of diversity among the results with 0 corresponding
-                to minimum diversity and 1 to maximum diversity.
-                Defaults to 0.3.
-            * summary_enabled: whether to generate summaries or not. Defaults to False.
-            * summary_response_lang: language to use for summary generation.
-            * summary_num_results: number of results to use for summary generation.
-            * summary_prompt_name: name of the prompt to use for summary generation.
+        mmr_k: number of results to fetch for MMR, defaults to 50
```

Comment:
it seems those parameters are not passed for init but as kwargs. is it what you want?

Reply:
yes good point. The number of parameters keeps growing, but I see your point - if it's listed in the docstring might as well add it to the actual thing.
Let me update with this change.

Reply:
updated!